source/docs/being-a-team/how-to-ask-effective-questions.rst

```diff
@@ -0,0 +1,95 @@
+In the end, this research will pay off in the long run, helping you to learn about topics relevant to your question, and often helping you to understand the roots of the problems you encounter.
+
+Here are some steps that you could take to research your problem:
+
+- **Checking Official Documentation** - If documentation or instructions are available, make sure to carefully examine the relevant sections of them.
+  Your problem may already be addressed, or may be caused by you misreading something.
+
+- **Google** - Simply search your problem.
+  You will be surprised by how often this solves the issue.
+  Try word the question in many different ways to find similar questions that may have already been asked.
+  Focus on including keywords about your problem and if it is about programming, include relevant error messages and codes.
+
+.. tip::
+    When looking for past documents or sections of textbooks, add "filetype:pdf" to the beginning of your search to filter the search results into only .pdf documents.
+
+- **YouTube** - There are many great resources on YouTube which can help with even obscure problems.
+
+- **Reddit and Discord** - The `FTC Subreddit <https://www.reddit.com/r/ftc/>`_ and `Discord Server <https://discord.gg/first-tech-challenge>`_ can also be useful places to find previous answers or to ask your own.
+  Try searching through the past questions with keywords relating to your own.
+
+- **Past Examples** - Try find past implementations of whatever problem you are trying to solve.
+  This step may take more time to find the example, and then decipher how someone else figured out the solution, but this process can be super valuable.
+  For programming, digging through another person's repository can be very insightful.
+  Likewise, for mechanical design, taking a look at a teams technical binder or CAD release.
+
+.. tip::
+    For programming questions, it can sometimes be helpful to find past implementations through `GitHub Code Search <https://github.com/search?type=code&auto_enroll=true>`_.
+    Even if you don't find the solution, examining other codebases can teach you a lot about general strategies and functions you may not have seen before.
+
+Be Specific
+-----------
+
+For others to be able to provide a good solution to your problem, they need the specifics.
+This means doing the following:
+
+1. **Break down the problem.**
+
+You should try to pinpoint the problem as much as possible.
+For example, when encountering robot crashes, try to collect as much data as possible around the times when your robot crashes.
+You may not understand this information yourself, but it may allow others to solve your issue.
+It can also be useful to explain why you're asking.
+It can be easy to get stuck on a specific solution to a problem when it would be better solved in an entirely different way; this is known as the `XY Problem <https://xyproblem.info/>`_.
```

Comment:
```suggestion
   You should try to pinpoint the problem as much as possible. For example, when encountering robot crashes, try to collect as much data as possible around the times when your robot crashes. You may not understand this information yourself, but it may allow others to solve your issue. It can also be useful to explain why you're asking. It can be easy to get stuck on a specific solution to a problem when it would be better solved in an entirely different way; this is known as the `XY Problem <https://xyproblem.info/>`_.
```
The paragraphs here should also be formatted as one line, but should also be indented to be properly contained within the list.